lib/pages/discover.dart

```diff
@@ -34,7 +35,14 @@ class _DiscoverDevicesState extends State<DiscoverDevices> {
           deviceList.add(addr.ip);
         });
       }
-    }).onDone(_refreshController.refreshCompleted);
+    }, onDone: () {
+      _refreshController.refreshCompleted();
+      if(deviceList.isEmpty) {
+        setState(() {
+          _isListEmpty = true;
+        });
+      }
```

Comment:
You also need to set `deviceList` to `false` if it is not empty.

Reply:
Oh yes I forgot about that. Added it now.